build_binaries.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+builds="ironic-api ironic-conductor ironic-dbsync"
+need_hooks="osprofiler keystoneauth1 ironic"
+
+# imports needd because they are modules that are loaded.
+hidden_imports="--hidden-import distutils --hidden-import eventlet.hubs.epolls --hidden-import eventlet.hubs.kqueue --hidden-import eventlet.hubs.selects --hidden-import osprofiler --hidden-import keystoneauth1.loading._plugins.identity.generic  --hidden-import keystoneauth1.loading._plugins.identity.v2 --hidden-import keystoneauth1.loading._plugins.identity.v3"
+python_paths=""
+additional_data="--add-data service-types.json:/os_service_types/data/"
+additional_hooks="--additional-hooks-dir=."
+
+# copy files we need to inject
+cp /usr/lib/python2.7/site-packages/os_service_types/data/service-types.json .
```

Comment:
doesn't seem robust :(

Reply:
It stinks, tbh.

Reply:
maybe use find? or maintain the file in the repo and copy it inside the image? This might be too cumbersome to maintain though